lib/tapioca/loaders/loader.rb

```diff
@@ -110,7 +112,7 @@ def load_engines_in_zeitwerk_mode
         autoloader = Zeitwerk::Loader.new
 
         engines.each do |engine|
-          engine.config.eager_load_paths.each do |path|
+          engine.config.all_eager_load_paths.each do |path|
```

Comment:
Isn't `all_eager_load_paths` a new method added in the referenced commit? Should we be doing the opposite instead?
- Still calls `eager_load_paths`
- For Rails 7.2, we assign `eager_load_paths` with `all_eager_load_paths`'s value

Reply:
I think doing the opposite has potential to get us into weird situations. The `eager_load_paths` method still exists in Rails 7.2, it just returns something different than before.

If we patch it and Rails itself uses it in a way we don't expect, it could lead to strange scenarios. By making sure the new method exists even for old versions, we don't have to worry about it.

Reply:
Ah sorry I forgot that Rails config doesn't need a static method definition, so I thought this would cause NoMethodError.